doc/releases/migration-guide-3.7.rst

```diff
@@ -390,6 +390,35 @@ LED Strip
 Sensors
 =======
 
+* The :dtcompatible:`meas,ms5607` sensor driver has been changed to support
+  either the MS5611 sensor too. :dtcompatible:`meas,ms5607` is now renamed to
+  :dtcompatible:`meas,ms5611`, in order to be more generic. The DT attribute ``chip``
```

Comment:
```suggestion
  also the MS5611 sensor. :dtcompatible:`meas,ms5607` is now renamed to
  :dtcompatible:`meas,ms56xx`, in order to be more generic. The DT attribute ``chip``
```
?

Reply:
Updated and will push once I figure out how to address the other PR comment. Thanks for pointing this out.